.gitignore

```diff
@@ -1,3 +1,4 @@
 /node_modules
 /dist
 .DS_Store
+/images
```

Comment:
I expect images should be included in repo otherwise someone looking at your repo will not have access to them